Code/GraphMol/hanoitest.cpp

```diff
@@ -578,14 +547,12 @@ void test4() {
     atomcomparefunctor3 ftor(&atoms.front(), *m);
```

Comment:
We should perhaps be consistent with the calls to std::vector::front versus std::vector::data